refactor(react-scheduler): add display name to views (#2134)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/day-view.md

Name | Type | Default | Description
-----|------|---------|------------

> name? | string | `Day` | The view unique identifier. Required if you use several `DayView` plugins.

Reviewer: ```suggestion
name? | string | `Day` | The view's unique identifier. Required if you use several `DayView` plugins.
```

> displayName? | string |  | The view name used in UI plugins. Default value is equal to `name`.

Reviewer: ```suggestion
displayName? | string |  | The view's name used in UI plugins. The default value is `name`.
```

intervalCount? | number | 1 | Multiplies the default view interval.

> cellDuration? | number | 30 | Specifies the cell duration in minutes.

Reviewer: ```suggestion
cellDuration? | number | 30 | Specifies the cell's duration in minutes.
```

Reviewer: BTW, how can a cell have a duration?

Reviewer 2: This is a Scheduler, one cell ~ 30 minutes by default.
